src/components/common/EntityInfo.vue

```diff
@@ -58,32 +60,33 @@ export default class EntityInfo extends Mixins(EnumMixin) {
   @Getter getFilingType!: FilingTypes
   @Getter isEntityType!: boolean
   @Getter isIncorporationFiling!: boolean
+  @Getter isRegistrationFiling!: boolean
 
   /** The entity application title.  */
-  private get entityTitle (): string {
+  get entityTitle (): string {
```

Comment:
There's no harm in getters being called externally (which we don't do anyway). The upside is that now VS Code doesn't say this getter is not used.